person.rb

```diff
@@ -20,6 +22,11 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals.push(rental)
+    rentals.person = self
+  end
```

Comment:
- [ ] The `add_rental` method takes book and date as parameters, the person itself should also be sent as a parameter to create the new rental achieving the 'has-many' association. Something like so: 

```ruby
  def add_rental(book, date)
    Rental.new(date, self, book)
  end
```

Please kindly fix that.